syntax = "proto3";

package test_grpc;


service TestGRpc {
    rpc Put (PutRequest) returns (PutReply) {
    }
}

message PutRequest {
    bytes str = 1;
}
message PutReply {
    bytes str = 1;
}
